packages/terra-overlay/src/Overlay.module.scss

```diff
@@ -11,7 +11,39 @@
     justify-content: center;
     outline: none;
     overflow: hidden;
-    z-index: 100;
+
+    .inner {
+      height: 200%;
```

Comment:
I think if this 200% is going to work, the parent needs to use the overflow: scroll;
The larger content is the means to prevent scrolling behind overflow that occurs on mobile devices by having it's own scrolling, just with hidden scrollbars. So if we want this fix we'll need updates.